src/main/java/com/github/cameltooling/lsp/internal/completion/CompletionResolverUtils.java

```diff
@@ -16,18 +16,31 @@
  */
 package com.github.cameltooling.lsp.internal.completion;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.camel.parser.helper.CamelXmlHelper;
 import org.eclipse.lsp4j.CompletionItem;
 import org.eclipse.lsp4j.Position;
 import org.eclipse.lsp4j.Range;
+import org.eclipse.lsp4j.TextDocumentItem;
 import org.eclipse.lsp4j.TextEdit;
+import org.w3c.dom.Node;
 
+import com.github.cameltooling.lsp.internal.instancemodel.CamelURIInstance;
 import com.github.cameltooling.lsp.internal.instancemodel.CamelUriElementInstance;
+import com.github.cameltooling.lsp.internal.instancemodel.PathParamURIInstance;
+import com.github.cameltooling.lsp.internal.parser.ParserXMLFileHelper;
 
 /**
  * @author lheinema
  */
 public class CompletionResolverUtils {
 	
+	private static final List<String> POSSIBLE_DIRECT_REFERENCE = Arrays.asList("direct","direct-vm", "seda", "vm", "amqp", "jms", "activemq", "broker");
```

Comment:
I think this attribute should be renamed now that there not only "direct" and "direct-vm"
maybe POSSIBLE_REFERENCES or POSSIBLE_CROSS_REFERENCE